root/static/scripts/release-editor/fields.js

```diff
@@ -130,18 +130,43 @@ class Track {
 
         // Convert stuff like 111 into 1:11
 
-        if (/^\d+$/.test(length) && (4 - lengthLength) <= 1) {
-            var minutes, seconds;
-
-            if (lengthLength === 3) minutes = length[0];
-            if (lengthLength === 4) minutes = length.slice(0, 2);
+        if (/^\d+$/.test(length)) {
+            var minutes = null, seconds = null, hours = null;
+
+            switch (lengthLength) {
+                case 3:
+                    minutes = length[0];
+                    break;
+                case 4:
+                    minutes = length.slice(0, 2);
+                    if (parseInt(minutes, 10) >= 60)  {
+                        minutes = minutes - 60;
+                        hours += 1;
+                        length = hours + ":" + minutes + ":" + seconds;
```

Comment:
`seconds` is `null` here. I strongly suggest removing the `if` block from the switch.